OG_Format.adoc

```diff
@@ -20,6 +20,21 @@ _Edited: Emma Slater – 24/08/2022_
 _Reviewed: data format harmonization working group – 24/04/2023_
 _Finalized and proposed: data format harmonization group – _
 
+////
+* [[Authors]]
+////
+== Authors
+
+* https://github.com/castelao[Guilherme Castelão], _Scripps Institution of Oceanography - UC San Diego, USA_
```

Comment:
I belive Gui's italics are due to the section above which is improperly formated. It will be removed in #147. I have made the requested changes to authorship